<?xml version="1.0"?>

<launch>
    <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
        <param name="max_iterations" value="10"/>
        <param name="fixed_frame" value="odom"/>
        <param name="base_frame" value="base_footprint"/>
        <param name="use_imu" value="true"/>
        <param name="use_odom" value="false"/>
        <param name="use_vel" value="true"/>
        <param name="stamped_vel" value="false" />
        <param name="publish_pose_stamped" value="true" />

        <remap from="/imu/data" to="/imu_data" />
        <remap from="/vel" to="/speed"/>
    </node>

    <node pkg="racecar_navigation" type="lsm_odom" name="lsm_odom" output="screen" />
</launch>